wrims_v2/wrimsv2_plugin/src/wrimsv2_plugin/reporttool/Report.java

```diff
@@ -198,12 +196,40 @@ public void doProcessing(IProgressMonitor monitor) {
 		monitor.worked(20);
 		
 		// open files 1 and file 2 and loop over to plot
-		Group dssGroupBase = Utils.opendss(scalars.get("FILE_BASE"));
-		Group dssGroupAlt = Utils.opendss(scalars.get("FILE_ALT"));
+		HecTimeSeries htsBase = new HecTimeSeries(scalars.get("FILE_BASE"));
```

Comment:
If there's a console output for messages, consider setting the DSS messaging level lower than the default, for example
		HecTimeSeries.setMessageLevel(3);

Reply:
per @hxiedwr "Right now, we don’t have console outputs but haven’t got any complaints in the past. So I think this is not urgent. "